src/main/java/com/neonlab/product/controller/DriverController.java

```diff
@@ -3,21 +3,35 @@
 import com.neonlab.common.annotations.Loggable;
 import com.neonlab.common.dto.ApiOutput;
 import com.neonlab.product.apis.AddDriverApi;
-import com.neonlab.product.dtos.AddDriverRequest;
-import com.neonlab.product.dtos.AddDriverResponse;
+import com.neonlab.product.apis.DeleteDriverApi;
+import com.neonlab.product.apis.UpdateDriverApi;
+import com.neonlab.product.dtos.DriverDto;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 
 @RestController
 @Loggable
+@RequestMapping("/v1/driver")
 public class DriverController {
 
     @Autowired
     private AddDriverApi addDriverApi;
-    @PostMapping("/addDriver")
-    public ApiOutput<AddDriverResponse> addDriver(@RequestBody AddDriverRequest addDriverRequest){
-        return addDriverApi.addDriver(addDriverRequest);
+    @Autowired
+    private DeleteDriverApi deleteDriverApi;
+    @Autowired
+    private UpdateDriverApi updateDriverApi;
```

Comment:
Spacing is an issue. 

Please refer some existing code. To learn about spacing.

cc- @shrawanindeshpande